src/ts/components/uicontainer.ts

```diff
@@ -143,13 +156,13 @@ export class UIContainer extends Container<UIContainerConfig> {
       name: 'touchend',
       handler: (e) => {
         if (!isUiShown) {
-          // Only if the UI is hidden, we prevent other actions (except for the first touch) and reveal the UI
-          // instead. The first touch is not prevented to let other listeners receive the event and trigger an
-          // initial action, e.g. the huge playback button can directly start playback instead of requiring a double
-          // tap which 1. reveals the UI and 2. starts playback.
-          if (isFirstTouch && !player.isPlaying()) {
```

Comment:
`isFirstTouch` flag was removed as it seems redundant. As we are inside an `if` that checks for `!isUiShown` and because we always call `showUi()` from inside this if, we will always just enter this code path on first touch after UI was hidden.